go.mod

```diff
@@ -24,3 +24,7 @@ require (
 	google.golang.org/grpc v1.40.1 // indirect
 	google.golang.org/protobuf v1.27.1 // indirect
 )
+require (
+	cloud.google.com/go/storage v1.14.0
```

Comment:
This seems unnecessary for this change?

Reply:
The conflict resolution may have been incorrect.